INSTALL.md

```diff
@@ -100,21 +101,21 @@ For details, see [System Requirements for oneDAL](https://software.intel.com/con
 
 8. Build oneDAL via command-line interface. Choose the appropriate commands based on the platform and the compiler you use:
```

Comment:
I'd add here that not only the platform and the compiler matter but also the interfaces (DAAL vs. oneAPI)

Reply:
fixed